src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -1202,7 +1202,7 @@ namespace winrt::TerminalApp::implementation
 
         const auto dispatchToElevatedWindow = ctrlPressed && !IsRunningElevated();
 
-        std::string sessionType;
+        const char* sessionType;
```

Comment:
Since this is a primitive value now, you need to assign a default value. E.g. `auto sessionType = "";` (this should infer `const char*`).